security/response-process.md

```diff
@@ -0,0 +1,254 @@
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Working as intended | Let the reporter know this is the intended behavior. If they think this behavior could be improved, they can file a feature request. Close the security issue. When responding with this assessment, the VMT responder should explain why this conclusion was reached, in case the original report was unclear and the VMT has unintentionally misunderstood the original report. |
+| Bug                 | Let the reporter know this is unwanted behavior but not a security issue, and ask them to refile this as a bug. Close the security issue.                                                                                                                                                                                                                              |
+| Feature request     | Let the reporter know this is the intended behavior. If they think this behavior could be improved, they can file a feature request. Close the security issue.                                                                                                                                                                                                         |
+| Vulnerability       | Let the reporter know that the VMT has confirmed this unwanted behavior creates a security issue. Proceed with the process.                                                                                                                                                                                                                                               |
+| Out of scope        | Let the reporter know that the issue is out of scope for OQS. If applicable, assist them in refiling the report upstream. See the following section for further actions.                                                                                                                                                                                                  |
+
+#### "Out of scope" assessments
+
+OQS packages cryptographic algorithm implementations from a variety of upstream sources.
+These vary widely in level of support and software development expertise.
+OQS assumes no obligation to fix vulnerabilities in upstream code, but may choose to do so on a best-effort basis.
```

Comment:
If OQS decides to fix upstream code on a best-effort basis (e.g., with a local patch) but the upstream remains unfixed, I think the process should outline the need for a warning about this in the security advisory text.

Reply:
@bhess, please mark resolved if in agreement with the changes by @SWilson4 .